Error while building on Alpine linux:

I'm trying to build antithesis jar files using following Dockerfile:

```
FROM alpine:3.20.3
RUN apk add --no-cache build-base gradle make openjdk21 patchelf swig

COPY gradle gradle
COPY ffi ffi
COPY sdk sdk
COPY gradlew* default.nix gradle.properties LICENSE lombok* README* settings.gradle shell.nix ./

#Without this line, error happened: compiler can't find a file jni_md.h
#ENV C_INCLUDE_PATH=/usr/lib/jvm/java-21-openjdk/include/linux/

RUN gradle build
```

and got the error:
```
10.36 # Create the wrapper DSO used by JNI to access libvoidstar
10.36 cc -I"/usr/lib/jvm/java-21-openjdk/include" -I . -shared -o libFfiWrapper.so *.c
10.36 In file included from FfiWrapper_wrap.c:168:
10.36 /usr/lib/jvm/java-21-openjdk/include/jni.h:45:10: fatal error: jni_md.h: No such file or directory
10.36    45 | #include "jni_md.h"
10.36       |          ^~~~~~~~~~
10.36 compilation terminated.
```


So, i have to add java-21-openjdk/include/linux directory into C_INCLUDE_PATH by my own, in the Dockerfile.

I think, this is a bug, and we have to add it into Makefile: https://github.com/antithesishq/antithesis-sdk-java/blob/main/ffi/swig/Makefile#L15

To add ${JAVA_HOME}/**include/linux** there:
`
${CC} -I"${JAVA_HOME}/include" -I"${JAVA_HOME}/include/linux" -I . -shared -o libFfiWrapper.so *.c
`

--- a/ffi/swig/Makefile
+++ b/ffi/swig/Makefile
@@ -12,7 +12,7 @@ build:
 
 build-jni:
 	# Create the wrapper DSO used by JNI to access libvoidstar
-	${CC} -I"${JAVA_HOME}/include" -I . -shared -o libFfiWrapper.so *.c
+	${CC} -I"${JAVA_HOME}/include" -I"${JAVA_HOME}/include/linux" -I . -shared -o libFfiWrapper.so *.c
 
 	# Add a dependency which ldd will try to resolve at runtime
 	# ldd will fail to resolve this outside of the Antithesis runtime environment

